android/java/proguard.flags

```diff
@@ -8,3 +8,26 @@
 -keep class org.chromium.chrome.browser.ChromeTabbedActivity {
     *** hideOverview(...);
 }
+
+-keep class org.chromium.components.browser_ui.site_settings.ContentSettingsResources {
+    private static *** getResourceItem(...);
```

Comment:
We should generalize it, as we don't care whether it's `private` or not. And if they change it, we will simply get crash on trying to invoke it.
Same applies to `static`, we should add unit test instead to check that it is still static, as they may change it and we will get crash on trying to invoke it.

Reply:
And we should also add tests that `non-static` functions remain `non-static`. As, if they change it in upstream, current tests will pass, but we will get crashes in runtime on trying to invoke them. This applies to `hideOverview` as well, we overlooked this moment before.